spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -1675,14 +1777,43 @@ deployer.<app>.kubernetes.imagePullPolicy=Always
 
 Replace `<app>` with the name of your application and `Always` with your desired image pull policy.
 
-You can configure an image pull policy at the server level in the container `env` section of a deployment YAML, as the following example shows:
+You can configure an image pull policy at the global server level, as the following example shows:
+
+For Streams:
 
 ====
-[source]
+[source,yaml]
```

Comment:
Here's a rendering of how we could avoid repetition of redundant properties.

![image](https://user-images.githubusercontent.com/1562654/51216915-e70f7d80-18da-11e9-9fe1-e5a5e9a51cef.png)